modules/mo_pytorch/mo_extensions/load/pytorch/loader.py

```diff
@@ -24,6 +24,8 @@
 from torch.autograd import Variable
 
 from extensions.load.loader import Loader
+from extensions.middle.ConvertGroupedStridedSlice import ConvertGroupedStridedSlice
+ConvertGroupedStridedSlice.enabled = False
```

Comment:
Can you please try to export model to ONNX and convert using MO? Interesting to see if we face similar problem.

Reply:
We can get IR model by this way, and it will pass the test

Reply:
Can you please try with the following code?

```python
# Comment
# from extensions.middle.ConvertGroupedStridedSlice import ConvertGroupedStridedSlice
# ConvertGroupedStridedSlice.enabled = False
```

```python
# front/pytorch/slice_ext.py

from extensions.middle.ConvertGroupedStridedSlice import ConvertGroupedStridedSlice

class StridedSliceFrontExtractor(FrontExtractorOp):
    op = 'StridedSlice'
    enabled = True

  def run_after(self):
    return [ConvertGroupedStridedSlice]
```

Reply:
The same problem

Reply:
Please add a note:
```python
from extensions.middle.ConvertGroupedStridedSlice import ConvertGroupedStridedSlice
# TODO: fix this bug
ConvertGroupedStridedSlice.enabled = False
```